homeassistant/components/intellidrive/coordinator.py

```diff
@@ -0,0 +1,48 @@
+"""Coordinator for Reisinger Intellidrive."""
+
+from datetime import timedelta
+import logging
+from typing import Any
+
+import reisingerdrive
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_TOKEN
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN, UPDATE_INTERVAL
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ReisingerCoordinator(DataUpdateCoordinator):
```

Comment:
Similar to the discussion about the name of the integration: If you plan to add additional reisinger integrations with support for more products, is this a bad choice of name?

Reply:
Yes you are right, changed that